packages/phisherman/src/lib/Phisherman.ts

```diff
@@ -0,0 +1,82 @@
+import { fetch, FetchMethods, FetchResultTypes, QueryError } from '@sapphire/fetch';
+import type { PhishermanReportType, PhishermanReturnType } from './PhishermanTypes';
+
+let storedApiKey: string;
+
+/**
+ * Checks if a link is detected as a scam or phishing link by phisherman.
+ * @param domain The domain to check.
+ * @since 1.0.0
+ */
+export async function checkDomain(domain: string, apiKey: string = storedApiKey) {
+	const result = await fetch<PhishermanReturnType>(
+		`https://api.phisherman.gg/v2/domains/check/${domain}`,
+		{
+			headers: {
+				'Content-Type': 'application/json',
+				Authorization: `Bearer ${apiKey}`
+			}
+		},
+		FetchResultTypes.JSON
+	);
+
+	return {
+		isScam: result.classification === 'safe' ? false : true,
+		...result
```

Comment:
shouldn't this be inverted?

Reply:
No...? if the classification is `'safe'` then it's not scam, so `isScam` is `false`.

Reply:
The lines not the logic

Reply:
oooh, yes I suppose that would be better just in case the API starts returning a property `isScam` as well.

Reply:
Welp the api doesn't the only way we check if it is scam in v2 is but checking for the "classification" so I added the property to make it easier rather than having an if check every time